pages/about/staff/_slug.vue

```diff
@@ -4,10 +4,12 @@
         <!-- no search on this page -->
         <nav-breadcrumb :title="page.entry.title" />
 
+
         <block-staff-detail
             class="staff-detail-block"
             :image="parsedImage"
             :staff-name="parsedStaffName"
+            :alternative-name="page.entry.alternativeName"
```

Comment:
remove alternative-name and pass 2 props one for full text and one for language

Reply:
```
            :alternative-full-name="parsedAlternativeFullName"
            :language="parsedLanguage"
```